Controllers/AccountController.cs

```diff
@@ -0,0 +1,148 @@
+﻿using HK2TProject_HotelManage_Server.Models.Authenticate;
+using HK2TProject_HotelManage_Server.Services.Users;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.Extensions.Configuration; //Thêm thư viện này
+
+
+namespace HK2TProject_HotelManage_Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        
+        private readonly IUserService _userService;
+
+        public AccountController(IUserService userService)
+        {
+            _userService = userService;
+        }
+
+     
+
+        [HttpPost("login")]
```

Comment:
ý tôi là you nên gộp 2 logic loginDEMO và loginMAIN thành 1 , fuc Login này chỉ giải quyết cho case đăng nhập thành công tạo token , còn ko thành công thì ? thêm else nữa